### PR TITLE
[docs][cloud] Smart Driver overview

### PR DESCRIPTION
**Pull request by ddhodge**
Add additional information to Connect applications page.
Add Smart Driver overview page  preview/drivers-orms/smart-drivers//

@netlify /preview/yugabyte-cloud/cloud-connect/connect-applications/

### CODE REVIEW

File: docs/content/preview/drivers-orms/smart-drivers.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+No.
+
+### Are there recommended settings for the maximum lifetime of a connection?
+
+Apart from directing connections to healthy nodes, smart driver connections are no different. Smart drivers require no special optimizations or modifications to your application's connection handling.
+
+## Further reading
+
+To learn more about the driver, refer to the [architecture documentation of Smart Drivers](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/smart-driver.md).
```
Comment (aishwarya24) [inline docs/content/preview/drivers-orms/smart-drivers.md:168]:
```suggestion
To learn more about the driver, refer to the [architecture documentation of smart drivers](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/smart-driver.md).
```

File: docs/content/preview/yugabyte-cloud/cloud-connect/connect-applications.md [hunk 2]
```diff
@@ -12,34 +12,50 @@ menu:
...
+- Configure network access
+- Download the cluster certificate
+
+### Network access
+
+To enable inbound network access from your application environment to a cluster, you need to add the IP addresses to the cluster [IP allow list](../../cloud-secure-clusters/add-connections).
+
+For best performance and security, use a [VPC network](../../cloud-basics/cloud-vpcs/) and deploy your application in a VPC that is peered with your cluster's VPC.
+
+To take advantage of smart driver load balancing features when connecting to clusters in YugabyteDB Managed, applications using smart drivers must be deployed in a VPC that has been peered with the cluster VPC. For more information on smart drivers and using smart drivers with YugabyteDB Managed, refer to [YugabyteDB smart drivers for YSQL](../../../drivers-orms/smart-drivers/)
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-connect/connect-applications.md:36]:
```suggestion
To take advantage of smart driver load balancing features when connecting to clusters in YugabyteDB Managed, applications using smart drivers must be deployed in a VPC that has been peered with the cluster VPC. For more information on smart drivers and using smart drivers with YugabyteDB Managed, refer to [YugabyteDB smart drivers for YSQL](../../../drivers-orms/smart-drivers/).
```

File: docs/content/preview/drivers-orms/_index.md [hunk 3]
```diff
@@ -13,16 +13,14 @@ menu:
     parent: develop
     weight: 570
 type: indexpage
+showRightNav: true
 ---
 
 The [Yugabyte Structured Query Language (YSQL) API](../api/ysql/) builds upon and extends a fork of the query layer from PostgreSQL, with the intent of supporting most PostgreSQL functionality. Client applications can use the [PostgreSQL drivers](https://www.postgresql.org/download/products/2-drivers-and-interfaces/) to read and write data into YugabyteDB databases. YSQL-compatible PostgreSQL drivers are listed in the compatibility matrix below.
 
-In addition to the compatible upstream PostgreSQL drivers, YugabyteDB also supports **smart drivers**, which extend the PostgreSQL drivers to enable client applications to connect to YugabyteDB clusters without the need for external load balancers. YugabyteDB smart drivers have the following features:
+## Smart drivers
```
Comment (aishwarya24) [inline docs/content/preview/drivers-orms/_index.md:21]: Change the driver titles wherever applicable on this page-
YugabyteDB PGX, Yugabyte psycopg2 and any others.

**Review by aishwarya24 — COMMENTED** [review]
On the individual language index pages too, the driver titles need to be changed.

File: docs/content/preview/drivers-orms/smart-drivers.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+## Using YugabyteDB smart drivers
+
+Yugabyte has developed the following smart drivers, available as open source software under the Apache 2.0 license.
+
+| GitHub project | Based on | Learn more |
+| :--- | :--- | :--- |
+| [YugabyteDB JDBC Driver for Java](https://github.com/yugabyte/pgjdbc) | PostgreSQL JDBC Driver | [Documentation](../java/yugabyte-jdbc/) |
+| [YugabyteDB PGX Driver for Go](https://github.com/yugabyte/pgx) | jackc/pgx | [Documentation](../go/yb-pgx/) |
+| [YugabyteDB Psycopg2 Driver for Python](https://github.com/yugabyte/psycopg2) | PostgreSQL psycopg2 | [Documentation](../python/yugabyte-psycopg2/) |
+| [YugabyteDB node-postgres Driver for Node.js](https://github.com/yugabyte/node-postgres) | node-postgres | [Documentation](../java/yugabyte-jdbc/) |
```
Comment (ashetkar) [inline docs/content/preview/drivers-orms/smart-drivers.md:60]: I think the right url will be available in a day or two when node.js smart driver docs are published. The smart driver was published last Friday.

**Pull request merged**